inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/model/KnowledgeBase.java

```diff
@@ -141,6 +146,13 @@
     @Column(nullable = false)
     private String basePrefix = IriConstants.INCEPTION_NAMESPACE;
     
+    /**
+     * A List of explicitly defined root concepts that can be used if auto detection takes too long
+     */
+    @ElementCollection(fetch = FetchType.EAGER)
+    @CollectionTable(name = "knowledgebase_explicitly_defined_root_concepts")
```

Comment:
Can we please use a shorter name here, e.g. "knowledgebase_root_class"? (we should switch from "concept" to "class" in the application - but that's a different story)